### DISCUSSION TITLE
Brew Errors trying to install "espeak-ng"

### DISCUSSION THREAD

**Discussion by fingerfunk** [Homebrew/discussions#1854] [Tap maintenance and Homebrew development]
Hello,
I have been working through the dependencies in hopes of installing espeak-ng and am getting "no previously deleted formula" and "no formula found in taps" for libpcaudio-dev and something called "waywardgeek.sonic". Brew Doctor says I'm clean and "brew update" did recently grab a new formula but the issue still occurs.  I'm on a Macbook Air Catalina 10.15.5 w/ Python 2.7.16 

I am not a coder but have been hanging OK in terminal thus far. 
Here is the epspeak-ng site: https://github.com/espeak-ng/espeak-ng/blob/master/docs/building.md#dependencies-1

and it seems these scripts are related https://github.com/anarchivist/homebrew-espeak-ng

I believe the pcaudiolib.rb script worked correctly. It's as if this dependency has changed(?) Here is a paste of terminal and appreciate any insight!

MacBook-Air:~ brt$ brew install libsonic-dev
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 4 casks.

Warning: No available formula or cask with the name "libsonic-dev". Did you mean libnice?
==> Searching for similarly named formulae...
This similarly named formula was found:
libnice
To install it, run:
  brew install libnice
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
MacBook-Air:~ brt$ 

PASTE from waywardgeek-sonic script:
MacBook-Air:~ brt$ brew install anarchivist/espeak-ng/waywardgeek-sonic
Error: anarchivist/espeak-ng/waywardgeek-sonic is a head-only formula.
To install it, run:
  brew install --HEAD anarchivist/espeak-ng/waywardgeek-sonic
MacBook-Air:~ brt$ brew install --HEAD anarchivist/espeak-ng/waywardgeek-sonic
==> Installing waywardgeek-sonic from anarchivist/espeak-ng
==> Cloning https://github.com/waywardgeek/sonic.git
Updating /Users/brt/Library/Caches/Homebrew/waywardgeek-sonic--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at ba33141 Merge pull request #23 from danielbair/sonic-mac
Error: An exception occurred within a child process:
  Utils::Inreplace::Error: inreplace failed
Makefile:
  expected replacement of "libsonic.so.$(LIB_TAG)" with "libsonic.$(LIB_TAG).dylib"
  expected replacement of "libsonic.so.0" with "libsonic.0.dylib"

**Comment by carlocab** [comment]
Try
```
brew install anarchivist/espeak-ng/pcaudiolib
brew install anarchivist/espeak-ng/waywardgeek-sonic
```
As for your errors installing `espeak-ng`, you'll need to address those at https://github.com/anarchivist/homebrew-espeak-ng. That's a problem with the `espeak-ng` formula which needs to be fixed there.

**Comment by fingerfunk** [comment]
Thanks and apologies for my late replay. 🙂  I have a proper-ish system now with Ubuntu (dual boot macbook air) rather than playing in macOS terminal so hoping for a better outcome.
________________________________
